packages/stripe/src/definition.ts

```
@@ -20,151 +21,183 @@ const defaultLogger: Logger = {
-      subscriptions.forEach((sub) => sub && uow.delete("subscription", sub.id));
-
-      return await uow.executeMutations();
+    deleteSubscriptionsByReferenceId(referenceId: string) {
+      return this.serviceTx(stripeSchema)
+        .retrieve((uow) =>
+          uow.find("subscription", (b) =>
+            b.whereIndex("idx_reference_id", (eb) => eb("referenceId", "=", referenceId)),
+          ),
+        )
+        .mutate(({ uow, retrieveResult: [subscriptions] }) => {
+          subscriptions.forEach((sub) => uow.delete("subscription", sub.id));
```

Biome (2.3.14)

[error] This callback passed to forEach() iterable method should not return a value.

Either remove this return or remove the returned value.


(lint/suspicious/useIterableCallbackReturn)